flutter/test/native_sdk_integration_test.dart

```diff
@@ -0,0 +1,160 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+import 'mocks.dart';
+import 'sentry_flutter_options_test.dart';
+
+void main() {
+  group('$NativeSdkIntegration', () {
+    setUp(() {
+      TestWidgetsFlutterBinding.ensureInitialized();
+    });
+
+    test('test default values', () async {
+      final channel = const MethodChannel('initNativeSdk');
+
+      String? methodName;
+      dynamic arguments;
+      channel.setMockMethodCallHandler((call) async {
+        methodName = call.method;
+        arguments = call.arguments;
+      });
+      var sut = Fixture().getSut(channel);
+
+      await sut.call(HubAdapter(), createOptions());
+
+      channel.setMethodCallHandler(null);
+
+      expect(methodName, 'initNativeSdk');
+      expect(arguments, <String, dynamic>{
+        'dsn': fakeDsn,
+        'debug': true,
+        'environment': null,
+        'release': null,
+        'enableAutoSessionTracking': true,
+        'enableNativeCrashHandling': true,
+        'attachStacktrace': true,
+        'attachThreads': false,
+        'autoSessionTrackingIntervalMillis': 30000,
+        'dist': null,
+        'integrations': [],
+        'packages': [
+          {'name': 'pub:sentry', 'version': '5.1.1'}
```

Comment:
this will fail when we bump the version every time I believe, better reading the version from `sdkVersion` property

Reply:
Good catch!